tests/app/templates/home.html

```diff
@@ -30,6 +30,11 @@
         The `face` icon from Material, with the renderer explicitly set and an extra class to set the size
     </p>
 
+    <p>
+        {% icon 'icons8-icons8-48' renderer='ImageRenderer' %}
+        The `icons8` icon from Icons8, with the renderer explicitly set
```

Comment:
So if we do not hardcode the `icons` subdir in ImageRenderer, this would become:
```
{% icon 'icons'icons8-icons8-48' renderer='ImageRenderer' %}
```

Reply:
This would be feasible, but in my opinion it defeats the DRY principle, as you would add `icons/` every time you use the tag.